tests/simulator/initialize/test_particles_init_1d.py

```
@@ -8,11 +8,10 @@
 import matplotlib
 from ddt import data, ddt, unpack
 from pyphare.core.box import Box1D
-from pyphare.cpp import cpp_lib
+from pyphare import cpp
```

Ruff (0.8.2)

`pyphare.cpp` imported but unused

Remove unused import: `pyphare.cpp`

(F401)